build.sbt

```diff
@@ -1,16 +1,18 @@
 val PureconfigVersion     = "0.10.2"
-val SparkVersion          = "2.4.4"
+val SparkVersion          = "2.4.3" //To be compatible with GCloud Dataproc latest image
+
 val CatsEffectVersion     = "2.0.0"
 val LogbackVersion        = "1.2.3"
 val Log4catsVersion       = "1.0.0"
+val HadoopVersion         = "2.9.2"
 
 import org.scalafmt.sbt.ScalafmtPlugin.autoImport._
 
 lazy val root = (project in file(".")).
   settings(
     inThisBuild(List(
-      organization := "org.47deg",
-      scalaVersion := "2.12.8"
+      organization := "org.fortysevendeg",
+      scalaVersion := "2.11.12"
```

Comment:
Scala 2.11 is about to be deprecated, what is stopping us to use Scala 2.12?

Reply:
Two reasons mainly: 
- The version of the connector with springml only works with Scala 2.11
- The available versions of Spark in Google Cloud are for Scala 2.11